app/validators/questionnaire_validator.py

```diff
@@ -121,6 +121,6 @@ def validate_white_spaces(self):
                     text.startswith(" ") or text.endswith(" ") or "  " in text
                 ):
                     self.add_error(
-                        error_messages.INVALID_WHITESPACE_FOUND,
+                        error_messages.INVALID_WHITESPACE_FOUND.format(text=text),
                         pointer=translatable_item.pointer,
```

Comment:
It should be added separately not part of the error message. i.e.
```py
error_messages.INVALID_WHITESPACE_FOUND,
pointer=translatable_item.pointer,
text=text
```

Reply:
Code refactored as above.